source/includes/_snapshots.md

```diff
@@ -281,6 +283,9 @@ Property | Default | Description
 ---------|---------|-------------
 `force`  |  false  | Specifies whether to force restoring the snapshot on a manager that already contains blueprints/deployments.
 `recreate_deployments_envs` | true | Specifies whether deployment environments should be created for restored deployments.
+`tenant_name` | true | In case of restoring a snapshot from a version prior to 4.0.0, it specifies the name of a new tenant that the snapshot will be restored into. Otherwise - not relevant.
+`restore_certificates` | true | Specifies whether to try and restore the certificates from the snapshot and use them to override the current Manager certificates (it only happens if the snapshot's certificates metadata do not match the Manager's certificates metadata). Useful when there are live agents from an old Manager (the one that created the snapshot) and we wish to control them with the current Manager. At the end of the execution the Manager will automatically reboot - unless the `no_reboot` property is `True`.
```

Comment:
the current Manager certificates ~(it only happens if~ **in the event that** the snapshot's certificates metadata ~do~ **does** not match the **new** Manager's certificates metadata. ~).~
Useful when there are live agents from ~an old~ **the** Manager ~(the one that created the snapshot) and we wish~ **from which the snapshot was created that you want to control** ~them~ with the current Manager. ~At the end of the~ **After** execution **,** the Manager will automatically reboot - unless the `no_reboot` property is `True`.